### ISSUE TITLE
ファイルストアの公開URLについて

### ISSUE THREAD

**Issue by utautattaro** [NIFTYCloud-mbaas/UserCommunity#721]
お世話になっています。

現在ncmb jssdkを利用してアプリを開発しています。
こちらアプリ内で生成されたデータをncmbのファイルストアにアップロードして、全てアーカイブできるような形のものを開発しています。

ファイルストアにアップロードした際、”https://mb.api.cloud.nifty.com/2013-09-01/applications/xxxxxxxx/publicFiles/name.png”のように公開URLが発行されるかと思いますが、こちらはncmb.File.upload()メソッドで指定したファイル名(name.png)以前のパス(publicFiles/まで)は以後新規アップロードしたファイル含め不変なものなのでしょうか？

比較的多いデータがあるため、いちいちncmbにアクセスして公開URLを取得するのではなく、アプリ内で絶対パスで指定できればとおもいご質問させていただきました。

よろしくおねがいします。

**Comment by moongift** [comment]
はい、共通です。xxxxxはNCMBの管理画面にアクセスした時のURLに明記されているアプリケーションID（と個人的に呼んでます）です。
